src/components/lib/VFileUpload/VFileUpload.vue

```diff
@@ -0,0 +1,82 @@
+<script lang="ts" setup>
+import { onBeforeUnmount, onMounted } from 'vue'
+
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+
+interface FileUploadProps {
+  placeholder: string
```

Comment:
Make `placeholder` optional, as we have a default value.

```suggestion
  placeholder?: string
```